tago/account/__init__.py

```diff
@@ -0,0 +1,60 @@
+import requests # Used to make HTTP requests
+import json # Used to parse JSON
+import os # Used to infer environment variables
+from socket import TagoRealTime
+from actions import Actions
+from analysis import Analysis
+from dashboards import Dashboards
+from devices import Devices
+from buckets import Buckets
+
+API_TAGO = os.environ.get('TAGO_SERVER') or 'https://api.tago.io'
+REALTIME = os.environ.get('TAGO_REALTIME') or 'https://realtime.tago.io'
+
+class Account:
+    def __init__(self, token):
+        self.token = token
+        # Not sure about this next line..the 'default headers part'		
```

Comment:
The default_header is correct and can be used as specified. This comment is not needed, kindly remove it.